### PR TITLE
Remove usage of self

### PR DESCRIPTION
**Pull request by jonaz**
@bradfitz we merged the last one a little bit to soon :)

### CODE REVIEW

File: connection.go [hunk 1]
```diff
@@ -347,6 +347,12 @@ func (conn *Connection) timeoutWorker(sp *sendPackage) {
 	conn.Unlock()
 }
 
+func (conn *Connection) GetNextEvent() chan Event {
```
Comment (bradfitz) [inline connection.go]: seems unrelated?

Comment (jonaz) [inline connection.go]: yeah its not even used any more. My intentions was to remove it i think.

**Comment by jonaz** [comment]
Damit. I had forgotten the "Build only if .travis.yml is present" config on travis...
